docs/self-managed/concepts/access-control/applications.md

```diff
@@ -19,9 +19,13 @@ See our documentation on [adding an application](/self-managed/identity/user-gui
 There are three types of applications in Identity:
 
 - Confidential
-- Machine-to-machine
+- Machine-to-machine (M2M)
 - Public
 
+Public and confidential applications are applications where a user can log in, like a custom Tasklist.
```

Comment:
Good opportunity to link to both Tasklist and service worker documentation in the case users aren't familiar.